models/public/cocosnet/accuracy-check.yml

```diff
@@ -0,0 +1,52 @@
+evaluations:
+  - name: cocosnet
+    module: custom_evaluators.cocosnet_evaluator.CocosnetEvaluator
+    module_config:
+      network_info:
+        correspondence: {}
+        generator:
+          adapter: cocosnet
+        verification_network:
+          additional_layers: ["InceptionV3/Logits/AvgPool_1a_8x8/AvgPool"]
```

Comment:
the output name corresponds specific classification model, is not it?
It make sense add comment which model should be used and that for different models this value can be different or just specify model path here

Reply:
Add information about classification model to cocosnet.md

Reply:
please also stay comment here